geniza/corpus/models.py

```diff
@@ -65,3 +71,161 @@ def __str__(self):
     def natural_key(self):
         return (self.language, self.script)
 
+
+class FragmentManager(models.Manager):
+
+    def get_by_natural_key(self, shelfmark):
+        return self.get(shelfmark=shelfmark)
+
+
+class Fragment(models.Model):
+    shelfmark = models.CharField(max_length=255, unique=True)
+    # multiple, semicolon-delimited values. Keeping as single-valued for now
+    old_shelfmarks = models.CharField(
+        'Historical Shelfmarks',
```

Comment:
worth adding a docstring?

Reply:
do you mean help text for the old shelfmarks field?

Reply:
sorry, hard to indicate on github! I meant for the `Fragment` class itself, since the other models had them.